package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@supabase/shared-types",
-  "version": "0.1.79",
+  "version": "0.1.78",
```

```suggestion
  "version": "0.1.80",
```

Would recommend bumping up the version as the previous one was already released to npm